lib/workers/repository/update/pr/changelog/gitlab/index.spec.ts

```diff
@@ -4,6 +4,7 @@ import { partial } from '../../../../../../../test/util';
 import * as semverVersioning from '../../../../../../modules/versioning/semver';
 import * as hostRules from '../../../../../../util/host-rules';
 import type { BranchUpgradeConfig } from '../../../../../types';
+import { GitLabChangeLogSource } from './source';
```

Comment:
This PR targets `main` so will result in a new version of `v36` when merged. Is this fix critical enough to backport to the old `v35` release?

I found these "rules" in our docs: [^1]

> ## Which Renovate versions are officially supported?
> 
> Only the latest version of Renovate is supported by the Renovate maintainers.
> The Renovate team only fixes bugs in an older version if:
> 
> - the hosted app needs to stay on that older major version for a short time, or
> - some critical bug needs to be fixed and the new major is blocked
> 
> If you're using the hosted app, you don't need to do anything, as the Renovate maintainers update the hosted app regularly.
> If you're self hosting Renovate, use the latest release if possible.

[^1]: [Renovate docs, which Renovate versions are officially supported?](https://docs.renovatebot.com/faq/#which-renovate-versions-are-officially-supported)

Reply:
@rarkins WDYT? v35 publishing is already active, so backport should be easy

Reply:
Yes, let's backport